templates/zerver/help/import-from-slack.md

```diff
@@ -1,149 +1,306 @@
+
+1. Follow [step 4](https://zulip.readthedocs.io/en/stable/production/install.html#step-4-configure-and-use)
+   of the guide for [installing a new Zulip
+   server](https://zulip.readthedocs.io/en/stable/production/install.html).
+
+{end_tabs}
+
+#### Import details
+
+Whether you are using Zulip Cloud or self-hosting Zulip, here are few notes to keep
+in mind about the import process:
+
+- Slack does not export organization settings, so you will need to [configure
+  the settings for your Zulip
+  organization](/help/getting-your-organization-started-with-zulip#customize-organization-settings).
+  This includes settings like [email
+  visibility](/help/restrict-visibility-of-email-addresses),
+  [message editing permissions](/help/configure-message-editing-and-deletion#configure-message-editing-and-deletion_1),
+  and [how users can join your organization](/help/restrict-account-creation).
+
+- Slack does not export user settings, so users in your organization may want to
+  [customize their account settings](/help/getting-started-with-zulip).
+
+- Slack's user roles are mapped to Zulip's [user
+  roles](/help/roles-and-permissions) in the following way:
+
+| Slack role              | Zulip role    |
+|-------------------------|---------------|
+| Workspace Primary Owner | Owner         |
+| Workspace Owner         | Owner         |
+| Workspace Admin         | Administrator |
+| Member                  | Member        |
+| Single Channel Guest    | Guest         |
+| Multi Channel Guest     | Guest         |
+| Channel creator         | none          |
+
+- Messages in threads are imported, but they are not explicitly marked as
+  being in a thread.
 
-```
-cd /home/zulip/deployments/current
-./scripts/stop-server
-./manage.py convert_slack_data /tmp/slack_data.zip --token <token> --output /tmp/converted_slack_data
-./manage.py import '' /tmp/converted_slack_data
-./scripts/start-server
-```
+- Message edit history and `@user joined #channel_name` messages are not imported.
 
-This could take several minutes to run, depending on how much data
-you're importing.  The server stop/restart is only necessary when
-importing on a server with minimal RAM, where an OOM kill might
-otherwise occur.
+## Clean up after the Slack export
 
-**Import options**
+Once your organization has been successfully imported in to Zulip, you should
+delete [the Slack app](https://api.slack.com/apps) that you created in order to
+[export your Slack data](#export-your-slack-data).  This will prevent the OAuth
+token from being used to access your Slack workspace in the future.
 
-The commands above create an imported organization on the root domain
-(`EXTERNAL_HOST`) of the Zulip installation. You can also import into a
-custom subdomain, e.g. if you already have an existing organization on the
-root domain. Replace the last line above with the following, after replacing
-`<subdomain>` with the desired subdomain.
+## Get your organization members started
```

Comment:
Perhaps make the heading a bit more general? There are a couple of key items in here that aren't really about the individual members:
* org settings
* integrations

Reply:
Yeah, that section expanded as I was working on it. Maybe just "Get your organization started with Zulip"?